rasa/telemetry.py

```diff
@@ -563,6 +563,27 @@ def toggle_telemetry_reporting(is_enabled: bool) -> None:
 
     rasa_utils.write_global_config_value(CONFIG_FILE_TELEMETRY_KEY, configuration)
 
+def filter_errors(event: Dict[Text, Any], _unused_hint: Optional[Dict[Text, Any]] = None) -> None:
+    """Filtering errors.
+
+    Args:
+        event: event to be logged to sentry
+        _unused_hint: some hinting information sent alongside of the event
+
+    Returns:
+        the event without any sensitive / PII data or `None` if the event should
```

Comment:
I would add add here "or`None` if the event constitutes an `ImportError` which should be discarded".